### ISSUE TITLE
Text and icon alignment inconsistent for mdi & images

### ISSUE THREAD

**Issue by viharm** [notclickable-jordan/starbase-80#29]
Hi, I have recently updated from 1.1.6 to 1.5.1 and noticed this issue. The alignment of the text and the icons in a tile is different if MDI icons are used and different if custom images are used.
![image](https://github.com/notclickable-jordan/starbase-80/assets/2663937/ec7bf945-dab8-4ffe-a424-83298fbae053)

The config blocks in both cases are identical.

Am I doing something wrong?

**Comment by notclickable-jordan** [comment]
Whoops, sorry about that!

Can you give me a fragment of your config.json so I can replicate it?

**Comment by viharm** [comment]
Here's the output source for the example with the image...
![image](https://github.com/notclickable-jordan/starbase-80/assets/2663937/2bb4953d-ac01-48da-8720-77e0335ed04f)

and here's the output source for the with the MDI icon...
![image](https://github.com/notclickable-jordan/starbase-80/assets/2663937/2cc9dbe8-8fd8-452e-9e07-539226183b91)

**Comment by viharm** [comment]
> Can you give me a fragment of your config.json so I can replicate it?
>

Here's the config block for the service that uses a custom image...

```json
      {
        "name": "Short Name",
        "uri": "http://service.url",
        "description": "Short description",
        "icon": "/Custom/mariadb.png",
        "iconColor": "#FFF",
        "iconBG": "#555",
        "iconBubble": true
      },
```
And here's one for a service that uses MDI icon...

```json
      {
        "name": "Another Service",
        "uri": "http://different.url",
        "description": "Similar description",
        "icon": "mdi-certificate",
        "iconColor": "#FFF",
        "iconBG": "#555",
        "iconBubble": true
      }
```

**Comment by viharm** [comment]
Dashboard icons behave similar to custom image icons. I believe there is an issue with the layout with MDI icons.

**Comment by notclickable-jordan** [comment]
Fixed in v1.5.2

**Comment by viharm** [comment]
The fix works. Thanks.
